src/Utils/request/uploadFile.ts

```diff
@@ -31,7 +33,14 @@ const uploadFile = async (
         } catch {
           error = xhr.responseText;
         }
-        Notification.BadRequest({ errs: error.errors });
+        handleHttpError(
+          new HTTPError({
+            message: "Client error",
```

Comment:
add translation.

Also you don't need to keep merging to latest unless there's merge conflicts.